### ISSUE TITLE
Redis crash error on k8s (minikube)

### ISSUE THREAD

**Issue by Curt-Park** [Curt-Park/url-shortener#6]
[Explore-logs-2023-02-11 16_15_01.txt](https://github.com/Curt-Park/url-shortener/files/10712850/Explore-logs-2023-02-11.16_15_01.txt)

From bottom to top:
```bash
2023-02-11T16:06:56+09:00	{"log":"argv[2]: '\"KjrDv4TVzB\"'\n","stream":"stdout","time":"2023-02-11T07:06:56.7429863Z"}
2023-02-11T16:06:56+09:00	{"log":"argv[1]: '\"MeubPl7m000\"'\n","stream":"stdout","time":"2023-02-11T07:06:56.7429843Z"}
2023-02-11T16:06:56+09:00	{"log":"argv[0]: '\"set\"'\n","stream":"stdout","time":"2023-02-11T07:06:56.742981717Z"}
2023-02-11T16:06:56+09:00	{"log":"id=53 addr=172.17.0.1:4325 laddr=172.17.0.11:6379 fd=27 name= age=50 idle=0 flags=N db=1 sub=0 psub=0 ssub=0 multi=-1 qbuf=48 qbuf-free=20426 argv-mem=24 multi-mem=0 rbs=1024 rbp=5 obl=0 oll=0 omem=0 tot-mem=22320 events=r cmd=set user=default redir=-1 resp=3\n","stream":"stdout","time":"2023-02-11T07:06:56.74297205Z"}
2023-02-11T16:06:56+09:00	{"log":"------ CURRENT CLIENT INFO ------\n","stream":"stdout","time":"2023-02-11T07:06:56.742970092Z"}
...
2023-02-11T16:06:56+09:00	{"log":"------ STACK TRACE ------\n","stream":"stdout","time":"2023-02-11T07:06:56.731606758Z"}
2023-02-11T16:06:56+09:00	{"log":"\n","stream":"stdout","time":"2023-02-11T07:06:56.73148055Z"}
2023-02-11T16:06:56+09:00	{"log":"1:M 11 Feb 2023 07:06:56.723 # Crashed running the instruction at: 0x40000742d2\n","stream":"stdout","time":"2023-02-11T07:06:56.724012008Z"}
2023-02-11T16:06:56+09:00	{"log":"1:M 11 Feb 2023 07:06:56.723 # Accessing address: 0xa0d35240a0d38\n","stream":"stdout","time":"2023-02-11T07:06:56.724009925Z"}
2023-02-11T16:06:56+09:00	{"log":"1:M 11 Feb 2023 07:06:56.723 # Redis 7.0.8 crashed by signal: 11, si_code: 1\n","stream":"stdout","time":"2023-02-11T07:06:56.724005258Z"}
2023-02-11T16:06:56+09:00	{"log":"=== REDIS BUG REPORT START: Cut \u0026 paste starting from here ===\n","stream":"stdout","time":"2023-02-11T07:06:56.723994883Z"}
```

**Comment by Curt-Park** [comment]
https://github.com/redis/redis/issues/10734#issuecomment-1128018035

**Comment by Curt-Park** [comment]
Fixed in #7
